### ISSUE TITLE
Language support for TSQL and PL/SQL

### ISSUE THREAD

**Issue by kbuyens-salesforce** [semgrep/semgrep#4175]
**Is your feature request related to a problem? Please describe.**
I'd would like to prevent SQL injection in stored procedures.

**Describe the solution you'd like**
It would be good if semgrep supports the TSQL and PL/SQL languages. This would allow us to help reason about SQL injection in stored procedures. 

**Describe alternatives you've considered**
Major static analysis tools do not support both TSQL and PL/SQL (e.g. checkmarx).  Writing a custom SAST is quite a lot of effort.

**Comment by aryx** [comment]
Is there a formal specification somewhere of the grammar of those TSQL and PL/SQL? Or even better
a tree-sitter grammar for those languages?

**Comment by kbuyens-salesforce** [comment]
I don't think so. This might come close: https://github.com/antlr/grammars-v4/tree/master/sql/tsql and https://github.com/antlr/grammars-v4/tree/master/sql/plsql
